### PR TITLE
html-basics

### PR DESCRIPTION
**Pull request by TrandafilTatiana**
Screen Shot 2024-12-19 at 4 25 44 PM

### CODE REVIEW

**Review by IvanCeban — CHANGES_REQUESTED** [review]
Great job!
Just 1 small improvement required

File: index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    <p>
+      In this post, let's focus on HTML. We will learn what HTML is all about,
+      and why you too should learn it.
+    </p>
+    <h3>What is HTML?</h3>
+    <p>
+      HTML stands for <span style="font-weight: bold">H</span>yper<span
+        style="font-weight: bold"
+        >T</span
+      >ext <b>M</b>arkup <strong>L</strong>anguage. It's a markup language that
```
[the remark this answers is not in the document]
Comment (TrandafilTatiana) [reply index.html:38]: Ok. I just wanted to show all cases)

**Review by IvanCebanPVT — APPROVED** [review]
Wrong account, sorry

**Comment by ciocoialexandru** [comment]
Looks awesome!

**Review by ciocoialexandru — APPROVED** [review]

**Review by IvanCeban — APPROVED** [review]

**Pull request merged**